specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/ChangeDataCapture.json

```diff
@@ -0,0 +1,410 @@
+{
```

Comment:
There are a bunch of breaking changes reported on this PR. Please get a breaking changes approval from Azure Breaking Changes Reviewers <azbreakchangereview@microsoft.com> and ARM can review the changes afterwards. Once you have the approval, please remove the "ARMChangesRequested" label from the PR to make the PR visible to the ARM reviewers.

Reply:
Hi,

This PR has been already reviewed in the repo: **azure-rest-api-specs-pr** on PR:[https://github.com/Azure/azure-rest-api-specs-pr/pull/11802]( https://github.com/Azure/azure-rest-api-specs-pr/pull/11802) and we have got approval for all the breaking changes.